lib/urldata.h

```diff
@@ -1755,6 +1755,7 @@ struct UserDefined {
   struct ssl_config_data proxy_ssl;  /* user defined SSL stuff for proxy */
 #endif
   struct ssl_general_config general_ssl; /* general user defined SSL stuff */
+  int ca_cache_timeout;  /* Certificate store cache timeout (seconds) */
   int dns_cache_timeout; /* DNS cache timeout (seconds) */
```

Comment:
Note, I put this here in `struct UserDefined`, next to `dns_cache_timeout`.

But maybe you'd prefer it in `struct ssl_general_config`?

I don't think it makes sense to put it in `struct ssl_config_data` because it doesn't have a different value for the proxy case.

Reply:
I decided it belonged in `struct ssl_general_config` so I moved it.